### ISSUE TITLE
GameBar showing time

### ISSUE THREAD

**Issue by NeverBeLazyG** [Maassoft/ColorControl#323]
first of all thank you very much for this very good tooll.

my question is: is there a way to increase the time Gamebar is shown ? it disappears to quick.

and second question: is there a way to display volume settnig in gamebar ?

thank you

**Comment by Maassoft** [comment]
No, currently the timeout is 5 seconds and then it will close. I can make this configurable in a new version.
The volume setting can not be added, I'll look if that's easily possible as well. I assume you mean the volume of the LG TV and not Windows?

**Comment by NeverBeLazyG** [comment]
> No, currently the timeout is 5 seconds and then it will close. I can make this configurable in a new version. 

my suggestion would be that you do it like this: as long as the mouse is over the gamebar, it stays on. when the mouse is moved out of the gamebar, it closes after the specified timeout. that way you could make the settings at your leisure without having to hurry.

>The volume setting can not be added, I'll look if that's easily possible as well. I assume you mean the volume of the LG TV and not Windows?

yes, i mean the volume of the tv

thank you
